### PR TITLE
Add AA01 successful submission test

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/support/page_objects/SubmissionConfirmationPage.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ class SubmissionConfirmationPage {
         return this;
     }
 
+    done() {
```
Comment (rsuller) [inline cypress/support/page_objects/SubmissionConfirmationPage.js]: done? 😂

File: cypress/support/page_objects/ConfirmChangeOfAccountingReferenceDate.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import ChangeOfAccountingReferenceDatePage from "../../support/page_objects/ChangeOfAccountingReferenceDatePage";
+
+class ConfirmChangeOfAccountingReferenceDate extends ChangeOfAccountingReferenceDatePage{
```
Comment (rsuller) [inline cypress/support/page_objects/ConfirmChangeOfAccountingReferenceDate.js]: Minor formatting here.

File: cypress/support/page_objects/SubmissionConfirmationPage.js (hunk 1 above)
Comment (danfalc) [reply cypress/support/page_objects/SubmissionConfirmationPage.js]: Inspiration taken from the other method in this file. `continue()`. Button simply says done

Comment (rsuller) [reply cypress/support/page_objects/SubmissionConfirmationPage.js]: I think we should have a base method and call it `jobDone()` 😄

Comment (danfalc) [reply cypress/support/page_objects/SubmissionConfirmationPage.js]: Haha. Should probably add button to the end of these methods but it's used in that many places would make it a huge change

**Pull request merged**